### PR TITLE
Change the label on known var to reflect use.

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Just a minor refactoring. 
Simplified the logic as vars not used

### CODE REVIEW

File: src/openrct2-ui/windows/RideConstruction.cpp [hunk 1]
```diff
@@ -2456,15 +2453,15 @@ static void sub_6CBCE2(
         // Set the temporary track element
         _tempTrackTileElement.SetType(TILE_ELEMENT_TYPE_TRACK);
         _tempTrackTileElement.SetDirection(trackDirection);
-        _tempTrackTileElement.AsTrack()->SetHasChain((edx & 0x10000) != 0);
+        _tempTrackTileElement.AsTrack()->SetHasChain((liftHillAndInvertedState & CONSTRUCTION_LIFT_HILL_SELECTED) != 0);
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/RideConstruction.cpp]: You changed the value here from 0x10000 to 0x1. Are all callers of this function also modified?

Comment (duncanspumpkin) [reply src/openrct2-ui/windows/RideConstruction.cpp]: yes thats the point of the change.

File: src/openrct2/ride/Ride.h [hunk 2]
```diff
@@ -993,6 +993,7 @@ struct rct_ride_properties
 #define TURN_MASK_4_PLUS_ELEMENTS 0xF800
 
 #define CONSTRUCTION_LIFT_HILL_SELECTED 1
+#define CONSTRUCTION_INVERTED_TRACK_SELECTED 2
```
Comment (tupaschoal) [inline src/openrct2/ride/Ride.h]: Why not a `constexpr int32_t` instead?

File: src/openrct2-ui/windows/RideConstruction.cpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2-ui/windows/RideConstruction.cpp]: I didn't see any changed callers to `sub_6CBCE2`, so I was wondering if it was complete.

File: src/openrct2/ride/Ride.h (hunk 2 above)
Comment (Gymnasiast) [reply src/openrct2/ride/Ride.h]: Should also use the `1 << 1` or `0b00000001` notation, I think.

File: src/openrct2-ui/windows/RideConstruction.cpp (hunk 1 above)
Comment (duncanspumpkin) [reply src/openrct2-ui/windows/RideConstruction.cpp]: `window_ride_construction_draw_track_piece` is the only caller and it used to stuff a load of unused information in that var as well. I've removed the unneeded information and removed the << 16.

File: src/openrct2/ride/Ride.h [hunk 3]
```diff
@@ -992,7 +992,8 @@ struct rct_ride_properties
 #define TURN_MASK_3_ELEMENTS 0x0700
 #define TURN_MASK_4_PLUS_ELEMENTS 0xF800
 
-#define CONSTRUCTION_LIFT_HILL_SELECTED 1
+constexpr uint32_t CONSTRUCTION_LIFT_HILL_SELECTED = 1 << 0;
+constexpr uint32_t CONSTRUCTION_INVERTED_TRACK_SELECTED = 1 << 1;
```
Comment (tupaschoal) [inline src/openrct2/ride/Ride.h]: Are they really unsigned? Because on all places they're being compared to `int32_t`

Comment (duncanspumpkin) [reply src/openrct2/ride/Ride.h]: I think we tend to use unsigned for flags for edge cases at 1 << 31

Comment (Gymnasiast) [reply src/openrct2/ride/Ride.h]: I think using unsigned for flags makes most sense. After all, there is no such thing as a "negative flag".

**Pull request merged**